### PR TITLE
nvuApp: logged user cannot load login or register page

### PR DESCRIPTION
**Pull request by qnislavnachev**
logged user cannot load login ot register page,
he need to logout first

### CODE REVIEW

**Comment by qnislavnachev** [comment]
logged user cannot load login or register page

**Comment by qnislavnachev** [comment]
PR is up to date

File: src/test/java/com/clouway/nvuapp/adapter/http/controllers/AvailableSessionHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import javax.servlet.http.Cookie;
+
+import static com.clouway.nvuapp.core.CustomMatchers.isRedirectingTo;
+import static org.junit.Assert.assertThat;
+
+public class AvailableSessionHandlerTest {
+    public JUnitRuleMockery context = new JUnitRuleMockery();
+
+    @Test
+    public void happyPath() throws Exception {
```
Comment (mgenov) [inline src/test/java/com/clouway/nvuapp/adapter/http/controllers/AvailableSessionHandlerTest.java]: Name looks misleading for the case when no session is available.

**Comment by qnislavnachev** [comment]
renamed test - now it is more understandable then happyPath

**Pull request merged**